src/main/java/ca/corefacility/bioinformatics/irida/service/AnalysisSubmissionService.java

```diff
@@ -253,40 +254,40 @@ public ProjectAnalysisSubmissionJoin shareAnalysisSubmissionWithProject(Analysis
 	 *
 	 * @param search      basic search string
 	 * @param name        analysis submission name
-	 * @param state       {@link AnalysisState} of the submission to search
+	 * @param states       Set of {@link AnalysisState} of the submission to search
 	 * @param workflowIds set of workflow UUIDs to search
 	 * @param project     {@link Project} to search in
 	 * @param pageRequest a {@link PageRequest} for the results to show
 	 * @return a page of {@link AnalysisSubmission}
 	 */
-	public Page<AnalysisSubmission> listSubmissionsForProject(String search, String name, AnalysisState state,
+	public Page<AnalysisSubmission> listSubmissionsForProject(String search, String name, Set<AnalysisState> states,
 			Set<UUID> workflowIds, Project project, PageRequest pageRequest);
 
 	/**
 	 * Get a page of all {@link AnalysisSubmission}s in the system
 	 *
 	 * @param search      basic search string
 	 * @param name        analysis submission name
-	 * @param state       {@link AnalysisState} of the submission to search
+	 * @param state       Set of {@link AnalysisState} of the submission to search
```

Comment:
`state` should be changed to `states` here as well

Reply:
:) Okay, updated in af95b2042393711b211308c7ae02dd921fa7fa59